src/main/java/org/folio/consortia/service/impl/UserAffiliationServiceImpl.java

```diff
@@ -26,12 +29,14 @@ public class UserAffiliationServiceImpl implements UserAffiliationService {
   private final UserTenantService userTenantService;
   private final TenantService tenantService;
   private final KafkaService kafkaService;
+  private final FolioExecutionContext folioExecutionContext;
   private final ObjectMapper objectMapper = new ObjectMapper();
 
   @Override
   @SneakyThrows
   @Transactional
   public void createPrimaryUserAffiliation(String eventPayload) {
+    FolioExecutionContext currentContext = (FolioExecutionContext) folioExecutionContext.getInstance();
```

Comment:
getInstance is a method used to save current folio context before executing code in same thread within another folio context. Don't see it's happening here. Why is it needed?

Reply:
I thought, I need to find current tenant. let me check to find context tenant without getInstance()

Reply:
folioExectionContext.getTenantId?

Reply:
thanks, fixed